### DISCUSSION TITLE
ImageMagick  Identify method for verifying the integrity of an image in the Magick.NET-Q8-AnyCPU Nuget Package> Signature Method

### DISCUSSION THREAD

**Discussion by stevensrf11** [dlemstra/Magick.NET#927] [Help]
I did not see any Identify method for verifying the integrity of an image in the Magick.NET-Q8-AnyCPU version 7.24.0 nuget package the I am using in a c# .Net Core 3.1 Window's Application?
I thought the  MagickImage class would have such a method.
Is there such a method in this package or is there and extension package I can obtain via a NuGet package that adds the method to the MagickImage class?

I saw a plugin  called DragonFly which is suppose to have an extension with the Identity method, but the plugin requires a lot of configuration to get it to work.
It appears not be just download if from NuGet and use it.
Has anyone used this plug to make use of the Identity method?
https://markevans.github.io/dragonfly/imagemagick

**Comment by stevensrf11** [comment]
I saw the MagckImage class as property named Signature,
Is this the answer to the question above?

public string Signature =>
    this._nativeInstance.Signature;
 Gets the signature of this image.

**Comment by dlemstra** [comment]
You can use the `.Ping` method of a `MagickImage` for this. But it is probably better to use the `MagickImageInfo` class.
